### ISSUE TITLE
Agenda for 3rd AllERCDevs (Asian / US Friendly) Tue, 2023-06-13 23:00 UTC

### ISSUE THREAD

**Issue by xinbenlv** [ercref/AllERCDevs#4]
The time of event will be 

*2023-06-13 UTC 2300-2359 Tuesday*, which translates to
 
- 19:00-19:59 on 2023-06-13 Tuesday in US Eastern Time.
- 07:00-07:59 on 2023-06-14 Wednesday in Singapore.
- 16:00-16:59 on 2023-06-13 Tuesday in US Pacific Time.

Or find in your timezone: https://www.timeanddate.com/worldclock/fixedtime.html?msg=3rd+AllERCDevs&iso=20230613T23&p1=1440&ah=1

The event will be hosted on Zoom, a link will be posted on "all-erc-devs" channel on this [Discord Server](https://discord.gg/2arW34S6J7)

Agenda will be updated and please feel free to comment to signup your agenda items.

### Main Session (1h)

1. [ ] (15-20min) *Featured Talk*: 
- [Web3js](https://github.com/web3/web3.js) by @gregthegreek
2. (10min) *Demo Show & Tell* (5-10min per slot)
- 5min [Karma](https://karmahq.xyz) by maheshmurthy@

4. (30min) *ERC Peer Review Requests* (5-10min per slot)
- [ERC-5269 ERC Detection and Discovery Interface](https://eips.ethereum.org/EIPS/eip-5269) by @xinbenlv 
- [ERC-7108 Clustered NFT](https://eips.ethereum.org/EIPS/eip-7108) by 
- @sullof 

5. Ad-hoc signup during event

### Social Chat for ERCers
What do you find interesting last week? It doesn't have to be related to ERC.

## Meeting Link

Zoom Meeting Link: https://us06web.zoom.us/j/88694529887?pwd=NDRWYXlhejBvcXFVSzEzRFluT0Jmdz09

Meeting ID: 886 9452 9887
Passcode: 338406

### TLDR
The text discusses a demo of Karma's reputation system for Web3 governance in DAOs, the challenges and opportunities in standardizing data and identity verification, the proposal of ERC 5269 for detecting interfaces and behaviors in smart contracts, and the introduction of ERC-7108 for creating clusters inside NFTs. It also mentions the idea of clustering token IDs for better management and recognition by marketplaces, and the Erc dev meeting's rotating schedule between different time zones.

### Full Transcription
https://app.airgram.io/7065496599555735553/my-meetings/0c8dc5900a3e11ee8dc50a2cbd030b4d

**Comment by sullof** [comment]
I can talk about ERC-6982 (Efficient Default Lockable NFT) and ERC-7108 (Clustered NFT). This time I won't miss it, I promise.

**Comment by sullof** [comment]
Sure. I can talk about ERC-7108.

**Comment by xinbenlv** [comment]
This is happening in ~1h
